orttraining/orttraining/core/graph/graph_augmenter.h

```diff
@@ -93,7 +101,8 @@ struct NodeDef {
   std::vector<ArgDef> output_args;
   NodeAttributes attributes;
   std::string name;
-};
+  int priority;
+};  // namespace training
```

Comment:
nit: namespace comment doesn't belong here

Reply:
Thanks, will change it.